x-pack/plugins/ml/public/application/settings/filter_lists/edit/edit_filter_list.js

```diff
@@ -355,7 +355,10 @@ export class EditFilterListUI extends Component {
               />
               <EuiFlexGroup justifyContent="flexEnd">
                 <EuiFlexItem grow={false}>
-                  <EuiButtonEmpty onClick={returnToFiltersList}>
+                  <EuiButtonEmpty
+                    onClick={returnToFiltersList}
+                    data-test-subj={'mlCancelFilterListButton'}
```

Comment:
To be consistent with other naming, I'd suggest to make this `mlFilterListCancelButton`.